components/automate-ui/src/app/pages/notifications/notifications.component.ts

```diff
@@ -194,7 +215,6 @@ export class NotificationsComponent implements OnInit {
   // successful? Seem it should be possible to update the local model with the
   // changes and have that trigger view updates?
   refreshRules() {
-    this.rules$ = this.service.fetchRules();
     this.rules$.subscribe(rules => {
```

Comment:
Since you are in here, this is a problem that needs addressing. Subscriptions should almost always be in ngOnInit only. This will result in multiple subscriptions to the same thing--that is a bad thing. You should be able to move this `this.rules$.subscribe` block into `ngOnInit` and get rid of `refreshRules` completely and everything should still work the same.